var/spack/repos/builtin/packages/r3d/package.py

```diff
@@ -36,6 +36,8 @@ class R3d(CMakePackage):
         description="Build R3D regression tests (versions 2019-04-24 or earlier)",
     )
 
+    variant("shared", default=False, description="Build position independent code")
```

Comment:
Nearly all packages that want to set the option below use `pic` for this variant (e.g., https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/netcdf-c/package.py).

Reply:
@tldahlgren done.